algosdk/v2client/indexer.py

```diff
@@ -495,19 +503,25 @@ def applications(
         return self.indexer_request("GET", req, query, **kwargs)
 
     def search_applications(
-            self, application_id=None, round=None, **kwargs):
+            self, application_id=None, round=None, limit=None, next_token=None, **kwargs):
         """
         Return applications that satisfy the conditions.
 
         Args:
             application_id (int, optional): restrict search to application index
             round (int, optional): restrict search to passed round
+            limit (int, optional): restrict number of results to limit
+            next_token (string, optional): used for pagination
```

Comment:
looks like the other functions call this `next_page`, we should keep it consistent